apiserversdk/util/http.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"math"
+	"net/http"
+	"time"
+)
+
+func RetryBackoff(attempt int, initBackoff time.Duration, backoffBase float64, maxBackoff time.Duration) time.Duration {
```

Comment:
nit: 
```suggestion
func GetRetryBackoff(attempt int, initBackoff time.Duration, backoffBase float64, maxBackoff time.Duration) time.Duration {
```
I prefer verb + noun